cmake/common.cmake

```diff
@@ -29,6 +29,35 @@ if(VERBOSE_LOGGING)
   set(TEST_HOST_LOGGING_LEVEL "debug")
 endif()
 
+option(LVI_MITIGATIONS "Enable LVI mitigations" ON)
+if(LVI_MITIGATIONS)
+  set(OE_TARGET_LIBC openenclave::oelibc-lvi-cfg)
+  set(OE_TARGET_ENCLAVE_AND_STD
```

Comment:
It would have been nice if the list was the same and the LVI path just decorated the paths, but cmake is probably exceptionally awkward for this.

Reply:
Since the targets are already such general abstractions, I was really hoping that `openenclave::oelibc` could just point at `liboelibc.a` or `liboelibc-lvi-cfg.a` depending on some other setting, but that's not how those targets are set up. I thought this was more readable (and searchable) that appending (sometimes-empty) suffixes everywhere.